tools/mo/openvino/tools/mo/moc_frontend/extractor.py

```diff
@@ -109,12 +114,16 @@ def try_get_nodes(input_model, node_name):
 
     # TODO: Add support for input/output group name and port index here (58562)
     # For new frontends logic shall be extended to additionally support input and output group names
-    idx = next((idx for idx, name in enumerate(found_node_names) if 'Tensor' in name), 0)
-    return found_nodes[idx]
+    return found_nodes[0]
 
 
-def fe_input_user_data_repack(input_model: InputModel, input_user_shapes: [None, list, dict, np.ndarray],
-                              freeze_placeholder: dict, framework: str, input_user_data_types=dict(), ):
+def fe_input_user_data_repack(
+    input_model: InputModel,
+    input_user_shapes: [None, list, dict, np.ndarray],
+    freeze_placeholder: dict,
+    framework: str,
+    input_user_data_types=dict(),
```

Comment:
Mutable default arguments in Python [are not recommended](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/), they can lead to very time-consuming bugs if the variable is being modified in the function body. Even if it is not in this case, it is dangerous for future code maintenance.

Reply:
That is right, it should be resolved at the beginning of the function's body. Use `None` as default argument and later on check for it and replace if necessary.

Reply:
Good point, I've not noticed that before, maybe because in this place changes were made by auto formatter. I've changed default argument to `None` , variable `input_user_data_types` is checked in line 205: `isinstance(input_user_data_types, dict)`